upstream/linux-fsl-sdk-v1.8.tar.bz2.sha1

```diff
@@ -0,0 +1 @@
+dce31beab234e1e326180605fc79c275798add13  /home/banther/projects/onie-repo/onie/build/download/linux-fsl-sdk-v1.8.tar.bz2
```

Comment:
remove the leading '/home/banther/projects/onie-repo/onie/build/download/' from the file -- otherwise this will only work for you in that directory.  compare to linux-fsl-sdk-v1.7.tar.bz2.sha1